clusterrunner_ha.sysconfig

```diff
@@ -0,0 +1,70 @@
+## Type:        string
+## Default:     "cluster-master.dev.box.net"
+## ServiceRestart: clusterrunner_ha
+#
+# The hostname of the proxy for the Cluster Master service.
+#
+CLUSTERMASTER_PROXY_HOSTNAME="master_host.domain.net"
+
+## Type:        string
+## ServiceRestart: clusterrunner_ha
+#
+# A comma-separated list of hostnames that will run the ClusterRunner
+# master service.
+#
+MASTER_HOSTS_CSV="master_host.domain.met"
```

Comment:
Because you likely won't be deploying ClusterRunner initially in HA-mode, you should just specify one hostname here.